lib/sharedTypes.js

```diff
@@ -54,6 +54,20 @@ export const ScheduleItemType = new GraphQLObjectType({
   },
 });
 
+export const SponsorItemType = new GraphQLObjectType({
+  name: 'SponsorItemType',
+  fields: {
+    websiteURL: {
+      type: GraphQLString,
+      description: 'The website address of the event sponsor',
+    },
+    imageURL: {
+      type: GraphQLString,
+      description: 'The logo url of the event sponsor',
+    },
+  },
```

Comment:
Surely you'd also need text field with the plain name of the sponsor? At the very least this can be used as image label text for a11y.